index.html

```diff
@@ -99,11 +103,13 @@
 </div>
 
     <div id="exifInformationHolder"></div>
+    <div id="imageContainer">
```

Comment:
Does this repo not use tags? Stylistically, I would expect tabs to delineate the container

Reply:
The formatting is somewhat inconsistent right now, in part due to a variety of authors trying to write quickly I suspect :)

Any change I make just here will still be inconsistent with the rest of the repo, so I didn't bother putting too much effort into it right now. If you have a particular way you think this could be formatted better I'm happy to make those changes.

If this tool is going to see continued development, it would help to adopt some style standards and use a linter for enforcing consistency (@everestpipkin for visibility). I'm not super familiar with git but I imagine there are open source linters & formatters that you can integrate with a repo?